tests/test_poll.py

```diff
@@ -59,7 +59,7 @@ def test_p4_error_command_line_wrong_port(stdout_checker: FuzzyCallChecker,
                             "-p4d", perforce_workspace.depot,
                             "-jtu", "https://localhost/?%s"])
     assert result != 0
-    stdout_checker.assert_has_calls_with_param("TCP connect to 127.0.0.1:1024 failed.")
+    stdout_checker.assert_has_calls_with_param("Connect to server failed; check $P4PORT")
```

Comment:
Why did the message change? Is it because the perforce server updated?

Reply:
The reason is the new error output in p4python 2024.1.0. I've added this description to the PR.
https://github.com/perforce/p4python/compare/2023.2.2...2024.1.0